pages/osx/apachectl.md

```diff
@@ -0,0 +1,15 @@
+# apachectl
+
+> Apache HTTP Server Control Interface for OSX.
```

Comment:
I'm not sure that `Control Interface` needs to be in title case. Perhaps lowercase would read better here?

Reply:
Fixed!